src/controllers/handlers/about-handler.ts

```diff
@@ -23,19 +23,17 @@ export async function aboutHandler(
   const maxUsers = await config.getNumber('MAX_USERS')
   const networkId = networkIds[ethNetwork]
 
-  const comms = await context.components.comms.getStatus()
-
-  const [lambdasHealth, contentStatus, lambdasStatus, resourcesOverload] = await Promise.all([
+  const [lambdasHealth, contentStatus, lambdasStatus, resourcesOverload, comms, realmName] = await Promise.all([
     status.getLambdasHealth(),
     status.getContentStatus(),
     status.getLambdasStatus(),
-    resourcesStatusCheck.areResourcesOverloaded()
+    resourcesStatusCheck.areResourcesOverloaded(),
+    context.components.comms.getStatus(),
+    realm.getName()
   ])
 
-  const realmName = await realm.getName()
-
   const healthy = lambdasHealth.lambdas && lambdasHealth.content && comms.healthy
-  const userCount = rpcSessions.sessions.size
+  const userCount = rpcSessions.sessions.size + (comms.usersCount || 0)
```

Comment:
Doesn't this count users twice?

Reply:
no, `comms.usersCount` is the total amount of peers in archipelago while `rpcSessions.sessions.size` is the total amount of peers connected to the BFF